src/main/java/seedu/ddd/model/AddressBook.java

```diff
@@ -109,6 +110,14 @@ public boolean hasEvent(Event event) {
         return events.contains(event);
     }
 
+    /**
+     * Returns true if an event with the same name as {@code event} exists in the address book.
+     */
+    public boolean hasEventOfName(Name eventName) {
```

Comment:
Nitpick, but I think I prefer `hasEventName` in order to follow the naming convention of the `hasClientId` and `hasVendorId` methods.